addons/stock/models/stock_move.py

```diff
@@ -983,7 +983,7 @@ def _create_extra_move(self):
                 precision_rounding=self.product_uom.rounding,
                 rounding_method ='UP')
             extra_move_vals = self._prepare_extra_move_vals(extra_move_quantity)
-            extra_move = self.copy(default=extra_move_vals)._action_confirm()
+            extra_move = self.with_context(skip_check=True).copy(default=extra_move_vals)._action_confirm()
```

Comment:
This should be in a separate commit with an explanation of the use case.

Reply:
All right.